src/test/java/Helloworld/HelloWorldTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class HelloWorldTest
+{
+	String s="hello  world";
```

Comment:
This string can be made a constant with `private static final` scope & modifiers.